libs/user-facing-errors/src/quaint.rs

```diff
@@ -175,6 +175,15 @@ pub fn render_quaint_error(kind: &ErrorKind, connection_info: &ConnectionInfo) -
                         .into(),
                 }))
             }
+            ConnectionInfo::Native(NativeConnectionInfo::Sqlite { file_path, db_name: _ }) => {
+                Some(KnownError::new(common::DatabaseOperationTimeout {
+                    time: "N/A".into(),
+                    context: format!(
+                        "Socket timeout (the database failed to respond to a query within the configured timeout — see https://pris.ly/d/sqlite-connector for more details.). Database: {}",
```

Comment:
There is no mention of any timeout in the docs, I guess something is missing. Could you create a docs issue?

Reply:
Good catch. Apparently we only mention `socket_timeout` for MySQL?
https://www.prisma.io/docs/orm/overview/databases/mysql#arguments

Reply:
And for PostgreSQL https://www.prisma.io/docs/orm/overview/databases/postgresql#arguments

Reply:
I don't think using word "socket" in here is correct - there is no sockets invovled at any point with native sqlite. Maybe just keep the text in the parenthesis only?

Reply:
SQLite connector uses this parameter to set the [busy timeout](https://www.sqlite.org/c3ref/busy_timeout.html).